src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.ts

```diff
@@ -796,6 +967,157 @@ export class InstructorHelpQuestionsSectionComponent extends InstructorHelpSecti
+      recipient: 'Bob',
+      recipientTeam: 'Team 2',
+      recipientEmail: 'bob@gmail.com',
+      recipientSection: '',
+      responseDetails: {
+        answer: [1, 1],
+        questionType: FeedbackQuestionType.RUBRIC,
+      } as FeedbackRubricResponseDetails,
+      instructorComments: [],
+    },
+    {
+      isMissingResponse: false,
+      responseId: '4',
+      giver: 'Bob',
+      giverTeam: 'Team 2',
+      giverEmail: 'bob@gmail.com',
+      giverSection: '',
+      recipient: 'Bob',
+      recipientTeam: 'Team 2',
+      recipientEmail: 'bob@gmail.com',
+      recipientSection: '',
+      responseDetails: {
+        answer: [1, 2],
+        questionType: FeedbackQuestionType.RUBRIC,
+      } as FeedbackRubricResponseDetails,
+      instructorComments: [],
+    },
+    {
+      isMissingResponse: false,
+      responseId: '5',
+      giver: 'Alice',
+      giverTeam: 'Team 1',
+      giverEmail: 'alice@gmail.com',
+      giverSection: '',
+      recipient: 'Emma',
+      recipientTeam: 'Team 2',
+      recipientEmail: 'emma@gmail.com',
+      recipientSection: '',
+      responseDetails: {
+        answer: [0, 1],
+        questionType: FeedbackQuestionType.RUBRIC,
+      } as FeedbackRubricResponseDetails,
+      instructorComments: [],
+    },
+    {
+      isMissingResponse: false,
+      responseId: '6',
+      giver: 'Charles',
+      giverTeam: 'Team 1',
+      giverEmail: 'charles@gmail.com',
+      giverSection: '',
+      recipient: 'Emma',
+      recipientTeam: 'Team 2',
+      recipientEmail: 'emma@gmail.com',
+      recipientSection: '',
+      responseDetails: {
+        answer: [2, 3],
+        questionType: FeedbackQuestionType.RUBRIC,
+      } as FeedbackRubricResponseDetails,
+      instructorComments: [],
+    },
+    {
+      isMissingResponse: false,
+      responseId: '7',
+      giver: 'David',
+      giverTeam: 'Team 1',
+      giverEmail: 'david@gmail.com',
+      giverSection: '',
+      recipient: 'Emma',
+      recipientTeam: 'Team 2',
+      recipientEmail: 'emma@gmail.com',
+      recipientSection: '',
+      responseDetails: {
+        answer: [1, 3],
+        questionType: FeedbackQuestionType.RUBRIC,
+      } as FeedbackRubricResponseDetails,
+      instructorComments: [],
+    },
+    {
+      isMissingResponse: false,
+      responseId: '8',
+      giver: 'Emma',
+      giverTeam: 'Team 2',
+      giverEmail: 'emma@gmail.com',
+      giverSection: '',
+      recipient: 'Emma',
+      recipientTeam: 'Team 2',
+      recipientEmail: 'emma@gmail.com',
+      recipientSection: '',
+      responseDetails: {
+        answer: [0, 1],
+        questionType: FeedbackQuestionType.RUBRIC,
+      } as FeedbackRubricResponseDetails,
+      instructorComments: [],
+    },
+  ];
+
+  readonly exampleRubricQuestionQuestionTabModel: QuestionTabModel = {
+    question: this.exampleRubricQuestionModel,
+    responses: this.exampleRubricQuestionResponseOutput,
+    statistics: '', // TODO: need JSON data string
```

Comment:
What is this TODO for? For question type except contribution question, the statistics field is designed to be empty